### PR TITLE
Remove private Infinispan API usage in the /debug servlet

### PR DESCRIPTION
**Pull request by rhusar**

### CODE REVIEW

File: clusterbench-ee10-web/src/main/java/org/jboss/test/clusterbench/web/debug/DebugServlet.java [hunk 1]
```diff
@@ -23,6 +22,9 @@
 @WebServlet(name = "DebugServlet", urlPatterns = { "/debug", "/debug/*" })
 public class DebugServlet extends AbstractCommonDebugServlet {
 
+    @Resource(lookup = "java:jboss/jgroups/channel/default")
+    private JChannel channel;
+
     @Resource(lookup = "java:jboss/infinispan/container/web")
     private EmbeddedCacheManager container;
```
Comment (pferraro) [inline clusterbench-ee10-web/src/main/java/org/jboss/test/clusterbench/web/debug/DebugServlet.java:29]: Why not get rid of this?

File: clusterbench-ee10-web/src/main/java/org/jboss/test/clusterbench/web/debug/DebugServlet.java [hunk 2]
```diff
@@ -31,17 +33,17 @@ public String getContainerSpecificDebugInfo(HttpServletRequest req) {
         StringBuilder info = new StringBuilder();
 
         // Get current cache nodes
-        info.append("Address: ").append(container.getAddress()).append(System.getProperty("line.separator"));
-        info.append("Coordinator: ").append(container.getCoordinator()).append(System.getProperty("line.separator"));
-        info.append("Members: ").append(container.getMembers()).append(System.getProperty("line.separator"));
+        info.append("Address: ").append(container.getAddress()).append(System.lineSeparator());
+        info.append("Coordinator: ").append(container.getCoordinator()).append(System.lineSeparator());
+        info.append("Members: ").append(container.getMembers()).append(System.lineSeparator());
```
Comment (pferraro) [inline clusterbench-ee10-web/src/main/java/org/jboss/test/clusterbench/web/debug/DebugServlet.java:38]: Why not use channel.getAddress(), channel.getView().getCoord(), channel.getView().getMembers()?

File: clusterbench-ee10-web/src/main/java/org/jboss/test/clusterbench/web/debug/DebugServlet.java [hunk 3]
```diff
@@ -31,17 +33,17 @@ public String getContainerSpecificDebugInfo(HttpServletRequest req) {
...
-        JGroupsTransport transport = (JGroupsTransport) container.getTransport();
-        for (Address infinispanWrapAddr : container.getMembers()) {
-            JGroupsAddress jgroupsWrapAddr = (JGroupsAddress) infinispanWrapAddr;
-            org.jgroups.Address physAddr = (org.jgroups.Address) transport.getChannel().down(new Event(Event.GET_PHYSICAL_ADDRESS, jgroupsWrapAddr.getJGroupsAddress()));
-            IpAddress ipAddr = (IpAddress) physAddr;
-            info.append(ipAddr.getIpAddress().getHostAddress()).append(":").append(((IpAddress) physAddr).getPort()).append("; ");
+        for (Address address : channel.getView().getMembers()) {
+            Address physicalAddress = (Address) channel.down(new Event(Event.GET_PHYSICAL_ADDRESS, address));
+            IpAddress ipAddress = (IpAddress) physicalAddress;
+            info.append(ipAddress.printIpAddress()).append("; ");
```
Comment (pferraro) [inline clusterbench-ee10-web/src/main/java/org/jboss/test/clusterbench/web/debug/DebugServlet.java:46]: Does this add much value?

Comment (rhusar) [reply clusterbench-ee10-web/src/main/java/org/jboss/test/clusterbench/web/debug/DebugServlet.java:46]: I was wondering the same thing if it's worth the hassle. @tommaso-borgato Does this ever help? Can we just get rid of this (listing physical addresses) altogether?

**Comment by honza-kasik** [comment]
Just adding reference to EAP issue https://issues.redhat.com/browse/JBEAP-26173

File: clusterbench-ee10-web/src/main/java/org/jboss/test/clusterbench/web/debug/DebugServlet.java (hunk 3 above)
Comment (honza-kasik) [reply clusterbench-ee10-web/src/main/java/org/jboss/test/clusterbench/web/debug/DebugServlet.java:46]: In tests we usually know physical addresses of individual nodes. We just assign a name to them and then check for that. But better get feedback from @tommaso-borgato as well.

Comment (tommaso-borgato) [reply clusterbench-ee10-web/src/main/java/org/jboss/test/clusterbench/web/debug/DebugServlet.java:46]: @rhusar I guess it's ok with me, we are just using `Node name:` and `Members:` when testing ranked routing tests.... `Physical addresses:` is not used AFAIK

**Review by tommaso-borgato — APPROVED** [review]

**Comment by rhusar** [comment]
As we have discussed, we want to clean this up substantially. Replaced by https://github.com/clusterbench/clusterbench/pull/419.

**Pull request closed without merging**